tests/foreman/ui/test_dashboard.py

```diff
@@ -638,13 +640,12 @@ def test_positive_content_host_subscription_status(self):
             'content-view-id': content_view.id,
             'lifecycle-environment-id': env.id,
             'activationkey-id': activation_key.id,
-        })
+        }, force_use_cdn=True)
```

Comment:
well, it actually causes the test to fail as cdn version won't start the agent up